src/neo/SmartContract/ApplicationEngine.Runtime.cs

```diff
@@ -143,7 +143,9 @@ internal void RuntimeLog(byte[] state)
         {
             if (state.Length > MaxNotificationSize) throw new ArgumentException();
             string message = Utility.StrictUTF8.GetString(state);
-            Log?.Invoke(this, new LogEventArgs(ScriptContainer, CurrentScriptHash, message));
+            var args = new LogEventArgs(ScriptContainer, CurrentScriptHash, message);
```

Comment:
Create args only if log or traceDebugSink are not null?

Reply:
will do